sabaibiometrics/management/commands/create_dummy_data.py

```diff
@@ -1,78 +1,120 @@
-                drug_allergy= "drug_allergy",
-                picture= "image/upload/v1715063294/ghynewr4gdhkuttombwc.jpg",
-            )
-            Visit.objects.create(
-                patient= Patient.objects.get(id=1),
-                date= "2021-01-01",
-                status= "status",
-            )
-            Vitals.objects.create(
-                visit= Visit.objects.get(id=1),
-                height= "100.00",
-                weight= "100.00",
-                systolic= 100,
-                diastolic= 100,
-                temperature= "37.00",
-                diabetes_mellitus= "Yes",
-                heart_rate= 100,
-                urine_test= "True",
-                hemocue_count= "100.00",
-                blood_glucose= "100.00",
-                left_eye_degree= "+4",
-                right_eye_degree= "+4",
-                left_eye_pinhole= "+23",
-                right_eye_pinhole= "+23",
-                others= "others",
-            )
-            Consult.objects.create(
-                visit= Visit.objects.get(id=1),
-                date= "2021-01-01",
-                doctor= User.objects.get(id=1),
-                past_medical_history= "past_medical_history",
-                consultation= "consultation",
-                plan= "plan",
-                referred_for= "referred_for",
-                referral_notes= "referral_notes",
-                remarks= "remarks",
-            )
-            Diagnosis.objects.create(
-                consult= Consult.objects.get(id=1),
-                details= "consult_details",
-                category= "consult_category",
-            )
-            Medication.objects.create(
-                medicine_name= "medicine_name",
-                quantity= 1,
-                notes= "notes",
-                remarks= "remarks",
-            )
-            Order.objects.create(
-                medicine= Medication.objects.get(id=1),
-                consult= Consult.objects.get(id=1),
-                quantity= 100,
-                notes= "order_notes",
-                remarks= "remarks",
-                order_status= "status",
-            )
-
-            self.stdout.write("Dummy data created successfully")
+            '''
+            Create a User for the Consult doctor if it doesn't exist. This ensures
+            that a doctor is always available for creating related 'consult' 
+            records; reliably referenced to the same doctor. 
+            '''
+
+            # Get first user available
+            doctor = CustomUser.objects.get(pk=1)
+
+            # Number of patients and visits to create
+            num_patients = 100
+            visits_per_patient = 3
+
+            for i in range(num_patients):
+                patient_number = i + 1
+
+                # Create a patient
+                patient = Patient.objects.create(
+                    village_prefix=["PC", "CA", "TT", "TK", "SV"][(patient_number - 1) // (num_patients // 5)],
```

Comment:
Just a small nitpick: Is there a reason for using this expression instead of 
`((patient_number + 1) % 5)`

Reply:
no reason actually, I just wanted to see the changes it makes on the output to make sure that the code was working and forgot to change back to the original one. but I believe it doesn't cause any issue, this function is just to generate dummy data to test the function for extracting data from postgres (it wont be used anywhere).